api/src/services/outputTemplates/outputTemplates.ts

```diff
@@ -16,6 +17,30 @@ export const outputTemplate: QueryResolvers['outputTemplate'] = ({ id }) => {
   })
 }
 
+export const getOrCreateOutputTemplate = async (outputTemplateInfo) => {
```

Comment:
Similar comment here but sort of applies to all of these -- would be nice to have a short comment doc about how these are meant to be called from the seed scripts only, just in case anyone gets the idea that they should be calling this somehow in code flow

Reply:
good idea- I realized that we can look at the `context` and figure out whether a request is coming from the graphql/lambda server and ensured that we have a hard-return in that case. Lmk if that looks good